src/Model/Behavior/SearchBehavior.php

```diff
@@ -15,11 +16,13 @@ class SearchBehavior extends Behavior
      * @var array
      */
     protected $_defaultConfig = [
+        'searchConfigurationMethod' => 'searchConfiguration',
```

Comment:
The key it a bit too long, we can use `searchConfigMethod`. Also please add info about the config option in the docblock.